pupil-spa/unsupported-browser-tests/index.spec.js

```diff
@@ -0,0 +1,26 @@
+const axios = require('axios')
+const testData = require('./browser-data.json')
+
+const req = async (userAgent) => {
+  const response = await axios({
+    method: 'get',
+    url: 'http://localhost:8181/',
```

Comment:
do we document all the ports in use anywhere? 🤔 
we might want to do that (in a separate ticket) as its useful

Reply:
Yes agreed.  I chose this one so it would not interfere with anything.